### ISSUE TITLE
Importing test library 'DebugLibrary' failed: AttributeError: StringIO instance has no attribute 'encoding'

### ISSUE THREAD

**Issue by tuminoid** [xyb/robotframework-debuglibrary#23]
Using Debian Jessie and robot is installed via pip. Python 2.7.9. After upgrading to robotframework-debuglibrary 0.8.2 (or 0.9.0), I started getting the following on every suite:

```
[ ERROR ] Error in file '/tests/robot/resources/keywords.txt': Importing test library 'DebugLibrary' failed: AttributeError: StringIO instance has no attribute 'encoding'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/DebugLibrary.py", line 64, in <module>
    COMMAND_LINE_ENCODING = sys.stdout.encoding or 'utf-8'
```

Line in question in `keywords.txt` is `Library        DebugLibrary`. There are no `Debug` used.

**Comment by xyb** [comment]
@tuminoid thanks for your report, I have released a new version 0.9.1 to fix this.

**Comment by tuminoid** [comment]
:+1: Thank you, great response time!
